triggers.md

```diff
@@ -96,6 +98,13 @@ The following example illustrates the basic idea:
 
 (The use of `<script>` is basically because only a few elements allow the HTML parser to enter RCDATA mode, and `<script>` has already been used in this way. This doesn't necessarily mean that these rules should respect the `script-src` CSP directive.)
 
+For use cases where it is preferable to provide speculation rule sets without modifying a document, we introduce the `Speculation-Rules` HTTP header. The header's values are URLs to external resources containing speculation rule sets. The rules are expressed in the same JSON object format as the inline case. The MIME type of the resource must be `application/speculationrules+json`.
+
+For example, suppose a rule set is available at `https://example.com/speculation-rules.json`. The response headers for a request for `https://example.com/` could include:
+```
+Speculation-Rules: speculation-rules.json
```

Comment:
I believe the current Chromium implementation uses a [structured field](https://www.rfc-editor.org/rfc/rfc8941.html), which would require this to be quoted. SF are generally preferred for new headers, and it gives us flexibility to extend the syntax in the future (e.g., with parameters).

Reply:
Done.